### DISCUSSION TITLE
tautulli

### DISCUSSION THREAD

**Discussion by vdias** [tteck/Proxmox#423] [Ideas]
Sugestion for new script...  Tautuli

https://tautulli.com/

**Comment by lounsberry** [comment]
I would also very much appreciate this!

**Comment by markwoodward86** [comment]
+1!

**Comment by ghost** [comment]
+1!

**Comment by darthrater78** [comment]
+1!

**Comment by tteck** [comment]
Ready for testing, please report your findings.
```
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/tautulli.sh)"
```

**Reply by sendmebits** [reply]
I ran into an issue where I was unable to enable SSL on the web interface for Tautulli. It was caused due to missing the "pyOpenSSL" module. I ran the below to resolve it:

`$ pip install pyopenssl`

**Reply by tteck** [reply]
Added to script

**Comment by sendmebits** [comment]
That was, quick thanks!!
